### ISSUE TITLE
Vidcoder 4.36 Settings not saving on exit?

### ISSUE THREAD

**Issue by Thermalzombie** [RandomEngy/VidCoder#574]
Hi been using vidcoder for ages now and as far as I can remember vidcoder has always saved my encode settings on exit.  So that when you load vidcoder again you can continue ripping a dvd or other task.

I have recently switched to windows 10 Pro and every time I load Vidcoder I have to fill in all the settings again I tried to save a profile and load that but that did not work either. 

Is there a checkbox somewhere to save on exit or somehting I am missing?

Thanks for reply

**Comment by RandomEngy** [comment]
No, there is no checkbox. It should definitely save right away. What version are you using? Installer, portable or Microsoft Store?

Are you using stable or Beta VidCoder?

**Comment by Thermalzombie** [comment]
Installer 32bit I created a profile in profile editor that seems to be working but changes made without clicking save profile dont remain when relaunching vidcoder.

When I was on windows 7 I could make changes then close relaunch vidcoder and my changes would restore.

**Comment by RandomEngy** [comment]
If you are using 32-bit you must be on an old version of VidCoder.

The settings should be stored in %appdata%\VidCoder\VidCoder.sqlite . If you do press "Save", do the changes stick around? Do changes inside Global Options stick around on relaunch? Does the sqlite file update its "last modified" time in Windows?

**Comment by Thermalzombie** [comment]
I am on 64bit sorry miss typed.
